test/verify/check-shell-host-switching

```diff
@@ -113,18 +113,12 @@ class TestHostSwitching(MachineCase, HostSwitcherHelpers):
     def setUp(self):
         super().setUp()
 
-        # Disable preloading on all machines ("machine1" is done in testlib.py)
-        # Preloading on machines with debug build can overload the browser and cause slowness and browser crashes
-        # In these tests we actually switch between machines in quick succession which can make things even worse
-        if self.is_devel_build():
-            for machine in ["machine2", "machine3"]:
-                self.disable_preload("packagekit", "playground", "systemd", machine=self.machines[machine])
-        # Also, quick logouts cause async preloads to run into "ReferenceError: cockpit is not defined"
+        self.setup_provisioned_hosts(disable_preload=True)
+        # Override hostname from machine1 to localhost
+        self.machines["machine1"].execute("hostnamectl set-hostname localhost")
```

Comment:
That's a  bit ugly now -- we could just change the test to assume "machine1" instead. But that can be a follow-up.

Reply:
Agreed, wanted to see if this was stable enough first.